### PR TITLE
chore(demo-playwright): `InputDate` add test click any day after `Until today`

### PR DESCRIPTION
**Pull request by mdlufy**
Add test for #9565

### CODE REVIEW

File: projects/demo-playwright/utils/page-objects/calendar.po.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import type {Locator} from '@playwright/test';
+
+export class TuiCalendarPO {
+    constructor(private readonly host: Locator) {}
+
+    public async getDays(): Promise<Locator[]> {
+        return this.host.locator('[automation-id="tui-primitive-calendar__cell"]').all();
+    }
+
+    public async clickOnCalendarDay(day: number): Promise<void> {
```
Comment (nsbarsukov) [inline projects/demo-playwright/utils/page-objects/calendar.po.ts:10]: Use already existing `TuiCalendarSheetPO`

https://github.com/taiga-family/taiga-ui/blob/d178edff4466a98b9183db9c8521afa0d17d66a5/projects/demo-playwright/utils/page-objects/calendar-sheet.po.ts#L3-L23

Comment (mdlufy) [reply projects/demo-playwright/utils/page-objects/calendar.po.ts]: used

File: projects/demo-playwright/utils/page-objects/input-date.po.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import type {Locator} from '@playwright/test';
+
+export class TuiInputDatePO {
+    public readonly textfield: Locator = this.host.getByRole('textbox');
+    public readonly calendar: Locator = this.host.page().locator('tui-calendar');
+    public readonly itemButton: Locator = this.host
+        .page()
+        .locator('tui-dropdown tui-calendar ~ * button');
```
[the remark this answers is not in the document]
Comment (mdlufy) [reply projects/demo-playwright/utils/page-objects/input-date.po.ts]: moved

File: projects/demo-playwright/tests/legacy/input-date/input-date.spec.ts [hunk 3]
```diff
@@ -61,10 +71,13 @@ test.describe('InputDate', () => {
...
-            input = api.apiPageExample.getByTestId(
-                'tui-primitive-textfield__native-input',
-            );
+            documentationPage = new TuiDocumentationPagePO(page);
+            example = documentationPage.apiPageExample;
+
+            inputDate = new TuiInputDatePO(example.locator('tui-input-date'));
+
+            calendar = new TuiCalendarPO(inputDate.calendar);
+            calendarSheet = new TuiCalendarSheetPO(inputDate.calendar);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/legacy/input-date/input-date.spec.ts:80]: Remove `calendarSheet`

File: projects/demo-playwright/tests/legacy/input-date/input-date.spec.ts [hunk 4]
```diff
@@ -92,60 +105,81 @@ test.describe('InputDate', () => {
...
         });
+
+        test('Click any day after `Until today` was selected', async ({page}) => {
+            await tuiGoto(page, `${DemoRoute.InputDate}/API?items$=1`);
+
+            await inputDate.textfield.click();
+            await calendar.itemButton.click();
+
+            await inputDate.textfield.click();
+            await calendarSheet.clickOnDay(1);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/legacy/input-date/input-date.spec.ts:179]:
```suggestion
            const [calendarSheet] = await calendar.getCalendarSheets();

            await calendarSheet.clickOnDay(1);
```

File: projects/demo-playwright/tests/legacy/input-date/input-date.spec.ts (hunk 3 above)
Comment (nsbarsukov) [reply projects/demo-playwright/tests/legacy/input-date/input-date.spec.ts:80]:
```ts
const [calendarSheet] = await calendar.getCalendarSheets();
await calendarSheet.clickOnDay(1);
```

File: projects/demo-playwright/tests/legacy/input-date/input-date.spec.ts (hunk 4 above)
Comment (mdlufy) [reply projects/demo-playwright/tests/legacy/input-date/input-date.spec.ts]: updated

File: projects/demo-playwright/tests/legacy/input-date/input-date.spec.ts (hunk 3 above)
Comment (mdlufy) [reply projects/demo-playwright/tests/legacy/input-date/input-date.spec.ts]: removed

**Review by nsbarsukov — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**